### PR TITLE
Hrajchert/website changes

### PR DESCRIPTION
**Pull request by hrajchert**
This PR:
* changes the social image
* changes some copy text 
* Adds a link to the TypeForm form
* Adds a GDPR banner, disabling youtube videos (which are not compliant when embedding), typeform and analytics until the user explicitly agrees.

Screen Shot 2021-09-22 at 14 38 32

Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [X] Commit sequence broadly makes sense
    - [X] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
    - [X] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [X] Self-reviewed the diff
    - [X] Useful pull request description
    - [X] Reviewer requested

### CODE REVIEW

File: marlowe-website/src/index.js [hunk 1]
```diff
@@ -153,7 +154,51 @@ function fixCrossLinks() {
...
+  });
+  $(".video-container p").each((_, elm) => {
+    elm.classList.add("hidden");
+    elm.classList.remove("flex");
+  });
+}
+function activateTypeform() {
+  var script = document.createElement("script");
+  script.type = "text/javascript";
+  script.async = !0;
```
Comment (jhbertra) [inline marlowe-website/src/index.js:170]: Isn't `!0` the same as `true`?

File: marlowe-website/src/index.js [hunk 2]
```diff
@@ -153,7 +154,51 @@ function fixCrossLinks() {
...
+  }
+  const gdprConsent = localStorage.getItem("gdpr-consent");
+  if (gdprConsent === "agree") {
+    activateComponentsWithCookies();
+    $("#gdpr-banner").addClass("hidden");
+  } else {
+    // Show the banner
+    $("#gdpr-banner").removeClass("hidden");
+    $(".agreeGDPR").on("click", () => {
+      localStorage.setItem("gdpr-consent", "agree");
```
Comment (jhbertra) [inline marlowe-website/src/index.js:190]: Should we also track if they've declined, and prevent the banner from popping up each time they visit? I think that might be more courteous.

**Review by jhbertra — COMMENTED** [review]
Just the one question about declining consent, otherwise looks good.

File: marlowe-website/src/index.js (hunk 1 above)
Comment (hrajchert) [reply marlowe-website/src/index.js:170]: I'd assume so... I just copy and pasted from the google analytics code. Maybe they used it to save some bytes as `!0` is smaller than `true`

File: marlowe-website/src/index.js (hunk 2 above)
Comment (hrajchert) [reply marlowe-website/src/index.js:190]: Well, two thougts:
* If they don't allow us to store stuff in localstorage/cookies, can we store the rejection there?
* currently we show the message every time you access the page as is the only way to see the message again (no button or dialog for managing GDPR)

**Pull request merged**